### ISSUE TITLE
Darken the slide area like Twitter Feature Request

### ISSUE THREAD

**Issue by kipYRN** [nisheeSheth/Custom_SliderView#2]
This is some awesome work! Can you darken the slid area just like twitter app? Also.. can the slider be moved from right to left?

**Comment by nisheeSheth** [comment]
yes, you can move silder right to left just you have to right this two line:-
first one line for icon if you have use then you can set on right side.
second one for move silder from right to left 
sliderMenu.rightSideIcon = true
sliderMenu.rigthsideSlider = true

**Comment by nisheeSheth** [comment]
you want to remove green color from silder view.?

**Comment by kipYRN** [comment]
No, I want it to have a shade

**Comment by nisheeSheth** [comment]
can do change in CustomSliderView. what ever you want to change.

**Comment by nisheeSheth** [comment]
or else you can give me image with kind of view you want

**Comment by kipYRN** [comment]
I'll try to do that, But if you can...kindly do it like this
![image](https://user-images.githubusercontent.com/39052357/41402562-41b8d6be-6fcb-11e8-8afe-638ee4404b2b.png)

**Comment by kipYRN** [comment]
Hey, did you figure out a way to make the hidden part dark?
